### DISCUSSION TITLE
Take advantage of Rspamd all 100% with integration of MTA

### DISCUSSION THREAD

**Discussion by Zitronet** [rspamd/rspamd#4232] [Q&A] [answered]
Hello everybody, first I want to say thank you for this great software

I have to choose between Postfix or Exim, but my main question is:

Does Rspamd work all features to all  100% with Postfix or Exim? 

By the way I already read this page https://rspamd.com/doc/integration.html but I want to confirm

**Comment by fatalbanana** [comment, marked as answer]
No, see [here](https://github.com/rspamd/rspamd/issues/3497#issuecomment-701937796) for a detailed look at limitations of Exim integration as compared to Postfix/Milter.
